docs/user_guide/datainterfaces.rst

```diff
@@ -45,35 +45,117 @@ but generally will not read the underlying data.
      the user of the required and optional input arguments to the downstream readers.
 
 
-3. Get metadata
-~~~~~~~~~~~~~~~
+3. Get and adjust metadata
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 Each ``DataInterface`` can extract relevant metadata from the source files and
-organize it for writing to NWB in a hierarchical dictionary. This dictionary
+organize it in a ``metadata`` hierarchical dictionary. This dictionary
 can be edited to include data not available in the source files.
 
 .. code-block:: python
 
-    metadata = spikeglx_interface.get_metadata()
-    metadata["NWBFile"]["experimenter"] = ["Darwin, Charles"]
+    metadata = interface.get_metadata()
+
+The DANDI Archive requires subject ID, sex, age, and species, which are rarely present in the source data. Here is how you would add them.
+
+.. code-block:: python
+
     metadata["Subject"] = dict(
         subject_id="M001",
         sex="M",
         age="P30D",
+        species="Mus musculus",
     )
 
+``subject_id`` is a unique identifier for the subject.
+
+``sex`` is the biological sex of the subject and can take the values:
+
+- ``M`` for Male
+- ``F`` for Female
+- ``U`` for Unknown
+- ``O`` for Other
+
+``age`` follows the `ISO 8601 duration format <https://en.wikipedia.org/wiki/ISO_8601#Durations>`_. For example, ``P30D`` is 30 days old, and ``P1Y`` would be 1 year old. To express a range of ages, you can use a slash, for example ``P30D/P35D`` for 30 to 35 days old.
+
+``species`` is the scientific Latin binomial name of the species. For example, ``Mus musculus``
+for a mouse.
+
+See :ref:`Subject Best Practices <best_practice_subject_exists>` for details
+
+The ``session_start_time`` is also required. This is sometimes found in the source data. If it is not found, you must add it.
+
+.. code-block:: python
+
+    from datetime import datetime
+    from dateutil import tz
+
+    metadata["NWBFile"]["session_start_time"] = datetime(2021, 1, 1, 12, 0, 0, tzinfo=tz.gettz("US/Pacific"))
+
+You can use ``tz.tzlocal()`` to get the local timezone.
+
+If the ``session_start_time`` is extracted from the source data, it is often missing a timezone. This is not required but is a recommended best practice. Here is how you would add it.
+
+.. code-block:: python
+
+    metadata["NWBFile"]["session_start_time"] = metadata["NWBFile"]["session_start_time"].replace(tzinfo=tz.gettz("US/Pacific"))
+
+NWB Best Practices also recommends several other fields that are rarely present in the extracted metadata. The metadata dictionary is the place to add this information.
+
+.. code-block:: python
+
+    metadata["NWBFile"].update(
+        session_id="session_1",
+        session_description="Observation of finches on the Galapagos Islands",
+        experiment_description="Observation of finches on the Galapagos Islands",
```

Comment:
```suggestion
        session_description="Observations of desert plants and reptiles on the island of San Cristobal.",
        experiment_description="Observations of wildlife across the Galapagos Islands.",
```

A user might wonder why these are duplicated - though we sometimes do that in practice, I think this could be an opportunity to highlight how they are different (with experiment being broader in scope than just this one session)